pkg/operator/operator.go

```diff
@@ -209,11 +210,27 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 		}
 	}
 
+	// Create a new cache to watch on Route objects from every namespace for the route metrics and the route status
+	// controller to share.
+	globalCache, err := cache.New(mgr.GetConfig(), cache.Options{
+		Scheme: mgr.GetScheme(),
+	})
+	if err != nil {
+		return nil, err
+	}
+	// Add the cache to the manager so that the cache is started along with the other runnables.
+	mgr.Add(globalCache)
```

Comment:
I called it `globalCache` because it includes every namespace. Wouldn't it be okay if other resources/controllers used it for cluster-scoped watches? Isn't that the point? Or am I misunderstanding something?

I don't mind changing it, but `routeCache` seems artificially limited, it's not just for routes, right? It could be for any object. is that what you are saying we want?

And `clusterScopedCache` is an overloaded term, aren't cluster scoped resources technically not apart of a namespace (when this cache watches all namespaces)?